app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,9 @@
+module Types
+	class UserType < Types::BaseObject
+		field :id, ID, null: false
+		field :first_name, String, null: false
+		field :last_name, String, null: false
+		field :email, String, null: false
+		field :google_token, String, null: false
+	end
```

Comment:
Was this the same module that @rer7891 wrote? It might be duplicated code.

Reply:
I think it is duplicated because it was pulled down from @tyladevon's branch. We might have to resolve a conflict in the next PR.